examples/rcn/mnist_small.py

```diff
@@ -0,0 +1,310 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py:percent
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+import os
+import time
+
+import matplotlib.pyplot as plt
+
+# %%
+# %matplotlib inline
+import numpy as np
+from jax import numpy as jnp
+from jax import tree_util
+from load_data import get_mnist_data_iters
+from preproc import Preproc
+
+from pgmax.fg import graph, groups
+
+os.environ["XLA_PYTHON_CLIENT_ALLOCATOR"] = "platform"
+os.environ["XLA_PYTHON_CLIENT_PREALLOCATE"] = "false"
+
+# %% [markdown]
+# ## 1. Load the data
+#
+#
+# %%
+hps, vps = 12, 12
+train_size = 20
+test_size = 20
+data_dir = "/storage/users/skushagra/MNIST/"
```

Comment:
This is supposed to be included as a tutorial example....People won't have access to `/storage`